Leanplum-SDK/Classes/LPRequestStorage.m

```diff
@@ -97,8 +97,8 @@ - (void)migrateRequests
  */
 - (void)saveRequests:(NSMutableArray *)requests
 {
-    if (requests.count > MAX_STORED_API_CALLS) {
-        NSRange range = NSMakeRange(0, requests.count - MAX_STORED_API_CALLS);
+    if (requests.count > MAX_EVENTS_PER_API_CALL) {
```

Comment:
I'm confused by this method. I was under the impression that we always store every individual event now immediately?